RecoEgamma/EgammaHLTProducers/plugins/EgammaHLTClusterShapeProducer.cc

```diff
@@ -153,6 +166,15 @@ void EgammaHLTClusterShapeProducer::produce(edm::StreamID sid,
     clshMap2.insert(recoecalcandref, sigmapp);
     clsh5x5Map2.insert(recoecalcandref, sigmapp5x5);
     clsh5x5NoiseCleanedMap2.insert(recoecalcandref, sigmapp5x5NoiseCleaned);
+
+    reco::CaloClusterPtr SCseed = recoecalcandref->superCluster()->seed();
+    const EcalRecHitCollection* rechits =
+        (std::abs(recoecalcandref->eta()) < 1.479) ? rechitsEB_.product() : rechitsEE_.product();
```

Comment:
Further up in is file there is a line:
```
if (EtaSC > 1.479)
```
should that be std::abs(EtaSC) is this variable is also a ```recoecalcandref->eta()```?

Reply:
this could be a bug;
but the `else {` loop where this potential bug lives, is not used anywhere in egamma HLT.
So even if it is a bug it can be ignored.
I can't confirm if it's a bug as I don't know what this is actually doing:
```
 if (EtaSC > 1.479)
        sigmaee = sigmaee - 0.02 * (EtaSC - 2.3);
    }
```

Reply:
Would it be OK to remove the ```else``` branch and put the ```then``` branch to be executed always?

Reply:
yes I think it's better to remove it as a cleanup. 
I'm talking to EGM POG just to make sure they agree.
I will get back to this.

Reply:
(After a chat with Riccardo S.) removed the else loop, as sigmaEtaEta etc is obsolete and not used anymore in favor of the better variable like sigmaIEtaIEta etc. 

@cms-sw/egamma-pog-l2  
@DebabrataBhowmik 
@ravindkv

Reply:
Could you please backport to 14_0?

Reply:
Do we need a backport? The main aim of this PR is to be able to perform phase2 HLT studies easily, using the 2 variables sMajor and sMinor. It's not that much needed for Run3 actually.

Reply:
According to the PPD schedule, 14_0 is to be used for the Phase-2 MC campaign:
https://docs.google.com/spreadsheets/d/1XqkT_HYs04ZbLKs7k4SyPZ9DcNfN7ZmHQxv2I_iI-Hw/edit#gid=402259138

Reply:
ah okay, thanks Martin. I will make a backport to 14_0